Helper/AlgoliaHelper.php

```diff
@@ -289,7 +289,7 @@ public function mergeSettings($indexName, $settings, $mergeSettingsFrom = '')
         } catch (\Exception $e) {
         }
 
-        $removes = ['slaves', 'replicas'];
+        $removes = ['slaves', 'replicas', 'decompoundedAttributes'];
```

Comment:
@mrahman3177 As this issue has been resolved with PHP API client v4, let's apply this fix to v3.13 only. (Do not merge forward to 3.14)